@here/harp-mapview/lib/PickHandler.ts

```diff
@@ -137,8 +134,8 @@ export class PickHandler {
      */
     intersectMapObjects(x: number, y: number): PickResult[] {
         const worldPos = this.mapView.getNormalizedScreenCoordinates(x, y);
-        const rayCaster = this.m_rayCaster;
 
+        const rayCaster = this.mapView.raycasterFromScreenPoint(x, y);
```

Comment:
Why is it necessary to create a new raycaster on every call?

Reply:
here the raycasterFromScreenPoint method returns only a link to the MapView.m_raycaster object